modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-preparing-playbook-machine-{context}"]
+= Preparing the machine to run the playbook
+
+Before you can add compute machines that use Red Hat Enterprise Linux as the
+operating system to a {product-title} {product-version} cluster, you must
+prepare a machine to run the playbook from. This machine is not part of the
+cluster but must be able to access it.
+
+.Procedure
+
+. Ensure that the `kubeconfig` file for the cluster and the installation program
+that you used to install the cluster are on the machine. One way to accomplish
+this is to use the same machine that you used to install the cluster.
+
+. Configure the machine to access all of the RHEL hosts that you plan to use as
+compute machines. You can use any method that your company allows, including a
+bastion with an SSH proxy or a VPN.
+
+. Configure a user on the machine that you run the playbook on that has SSH
+access to all of the RHEL hosts.
++
+[IMPORTANT]
+====
+If you use SSH key-based authentication, you must manage the key with an
+SSH agent.
+====
+
+. Install the OpenShift Command-line Interface (CLI), commonly known as `oc`, on
+the machine.
+
```

Comment:
# yum install openshift-clients
better to tell customer which package to install

Reply:
I'm moving this to the prereq - we have a separate module for installing `oc`. It's part of the cluster installation process.